app/routes.js

```diff
@@ -45,17 +45,22 @@ var BreadcrumbMaker = require('../lib/js/breadcrumb_maker.js');
 
     var globPage = glob(directory + "/**/" + basename + ".html");
 
-    globPage.then(function (file){
+    globPage.then(function (file) {
       var filePath = file[0];
       return filePath;
     }).then(function(filePath) {
       readFile(filePath).then(function(data) {
         content = data.toString();
-        var breadcrumb = breadcrumbMaker.getBreadcrumbForContent(url);
-        var taxons = getTaxons(url);
-
-        console.log("Breadcrumb", breadcrumb);
         var whitehall = filePath.match(/whitehall/);
+        var html_publication = content.match(/html-publications-show/);
+
+        if (html_publication) {
+          // Skip breadcrumbs and taxons for HTML publications since they have a unique format
+          console.log("Skipped breadcrumbs and taxons for HTML publication");
```

Comment:
Ah I see, I've removed the other log too. The if statement is now there purely to skip getting breadcrumbs and taxons for HTML publications since they come back as undefined.